modules/core/src/webgl/vertex-array.js

```diff
@@ -133,6 +135,7 @@ export default class VertexArray {
   // Must be a Buffer bound to GL.ELEMENT_ARRAY_BUFFER. Constants not supported
   setElementBuffer(elementBuffer = null, accessor = {}) {
     this.elements = elementBuffer; // Save value for debugging
+    this.elementsAccessor = accessor;
```

Comment:
For symmetry reasons with `setAttributes`, should we add support for the `setElementBuffer(accessor)` signature (in addition to `setElementBuffer(buffer)`?

(Almost makes me want to change the name of this function to just `setElements`, but not worth it now).